### PR TITLE
Fix docs for new site

### PR DESCRIPTION
**Pull request by renancaraujo**
# Description

This adapts the current markdown docs structure to be digetable by the site in order to generate the docs section (https://flame-engine.org/docs).

## Type of change

- [x] This change requires a documentation update - actually it is a doc update

### CODE REVIEW

**Review by spydon — APPROVED** [review]

File: doc/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<!-- {ignore} -->
+## ⚠️ Warning:
+This is a source file for the docs site and may not be up to date with the latest version.
```
Comment (luanpotter) [inline doc/README.md:3]: is this supposed to show on github only?
how come is github out-of-date? if anything can be out of date it's the site...
I think we should support viewing in both platforms

File: doc/audio.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+<!-- {ignore} -->
+## ⚠️ Warning:
+This is a source file for the docs site and may not be up to date with the latest version.
+Prefer to read it on [The official docs site](https://flame-engine.org/docs)
+<!-- {/ignore} -->
+
```
Comment (luanpotter) [inline doc/audio.md:6]: yeah I dont think we should add that to every file

File: doc/components.md [hunk 3]
```diff
@@ -72,18 +81,18 @@ This is available on all `PositionComponent` objects, and is especially useful o
 The most commonly used implementation of `PositionComponent` is `SpriteComponent`, and it can be created with a `Sprite`:
 
 ```dart
-    import 'package:flame/components/component.dart';
+import 'package:flame/components/component.dart';
 
-    Sprite sprite = Sprite('player.png');
+Sprite sprite = Sprite('player.png');
```
Comment (luanpotter) [inline doc/components.md:86]: I think this is wrong? cc @erickzanardo

**Review by luanpotter — COMMENTED** [review]
I am ok with the formatting changes
And ok with splitting the readme/summary
But that warning is weird to me.

File: doc/README.md (hunk 1 above)
Comment (erickzanardo) [reply doc/README.md:3]: I think the disclaimer is valid, but the message is confusing. It should say that this source can contain unpublished features, or something like that

File: doc/audio.md (hunk 2 above)
Comment (erickzanardo) [reply doc/audio.md:6]: I agree, only on the README is enough

File: doc/components.md (hunk 3 above)
Comment (erickzanardo) [reply doc/components.md:86]: yes, I have fixed that on #544

File: doc/README.md (hunk 1 above)
Comment (luanpotter) [reply doc/README.md:3]: if we get rid of develop, we can remove this message then right?

Comment (spydon) [reply doc/README.md:3]: Master will still be ahead of the last published version, so I think the message should say something like:
> This is a source file for the docs site and might contain documentation about features or changes that are not yet released.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -13,12 +13,11 @@ Please delete options that are not relevant.
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-# Checklist:
+## Checklist:
```
Comment (luanpotter) [inline .github/pull_request_template.md:16]: is this inside the description block? is everything a description?

File: FAQ.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ We have a few examples! The simplest of all is in [the root example folder](/exa
 However there is an [examples folder inside the doc folder](/doc/examples) that contains many more examples. Most newer features we are adding we are trying to add a specific example for that. Those are directed to specific features.
 
 If you want a more full-fledged game, please check [BGUG](https://github.com/fireslime/bgug), FireSlime's open source fast paced side-scrolling platformer. It is mostly up-to-date with Flame and uses a good chunck of the featues offered by the engine. Other examples game are [feroult](https://github.com/feroult)'s [Haunt](https://github.com/feroult/haunt), which is outdated but showcases Box2D,[renancaraujo](https://github.com/renancaraujo)'s port of the [trex](https://github.com/flame-engine/trex-flame) chrome game in Flame, and also [Bob Box](https://github.com/fireslime/bounce_box), which is easy to grasp and a good display of our core features.
-
+https://github.com/flame-engine/flame_forge2d/tree/master/example
```
Comment (luanpotter) [inline FAQ.md:42]: ?

File: README-PL.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ _Jest to nieoficjalne tłumaczenie. W chwili tłumaczenia wersja v0.22.0._
 
 <img src="https://i.imgur.com/vFDilXT.png" width="400">
 
-[English](https://github.com/flame-engine/flame) | [简体中文](README-ZH.md) | [Polski](https://github.com/mbiesiad/flame/blob/pl_PL/README-PL.md)
+[English](README.md) | [简体中文](README-ZH.md) | [Polski](README-PL.md) | [Русский](README-RU.md)
```
Comment (luanpotter) [inline README-PL.md:7]: `../RADME.md`

File: README-RU.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+[![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
```
Comment (luanpotter) [inline README-RU.md:1]: ?

**Review by luanpotter — APPROVED** [review]
LGTM

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**